corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -242,7 +242,7 @@
 
         COMMCAREHQ.app_manager.commcareVersion.subscribe(function () {
             $('.commcare-feature').each(function () {
-                var version = '' + $(this).data('since-version') || '1.1',
+                var version = '' + $(this).attr('data-since-version') || '1.1',
```

Comment:
Ah, I see you changed it because it was being converted to a number and loosing the `0`. This solution seems fine but then I would add a comment stating why using `data` is problematic.